.gitignore

```diff
@@ -53,4 +53,6 @@ ts-gen
 # Used to accelerate builds after sync
 patches/mtime-cache.json
 
-spec/fixtures/logo.png
\ No newline at end of file
+spec/fixtures/logo.png
+
+build/custom_args/*
```

Comment:
Maybe this can be added to your local `.git/info/exclude` and not included in the tree.